main/main.c

```diff
@@ -342,6 +343,8 @@ main(int argc, char **argv)
 	if (gatekeeper_logtype < 0)
 		rte_exit(EXIT_FAILURE, "Error registering gatekeeper log type\n");
 
+	RTE_VERIFY(prctl(PR_SET_KEEPCAPS, 1, 0, 0, 0) == 0);
```

Comment:
Let's add the following comment for `prctl()`:
`/* rte_eal_init(), which is called next, creates all threads that Gatekeeper will use before the code finds out if Gatekeeper is going to run as root or not. When Gatekeeper runs with a non-root user, Gatekeeper has to assign capabilities to the functional blocks, which, in turn, is only possible if the flag PR_SET_KEEPCAPS is already set. */`
Break this comment on as many lines as needed.